temporalio/converter.py

```diff
@@ -843,6 +844,10 @@ def _error_to_failure(
                 failure.application_failure_info.details.CopyFrom(
                     payload_converter.to_payloads_wrapper(error.details)
                 )
+            if error.next_retry_delay:
+                delay = google.protobuf.duration_pb2.Duration()
+                delay.FromTimedelta(error.next_retry_delay)
+                failure.application_failure_info.next_retry_delay.CopyFrom(delay)
```

Comment:
```suggestion
                failure.application_failure_info.next_retry_delay.FromTimedelta(error.next_retry_delay)
```

I wonder if this code works instead of a create-then-copy

Reply:
Thanks, yes it seems to.